src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
 
+function roundedArea(area:number):number {
+  return Math.floor(area * 100) / 100;
+}
+
+function checkArg(arg: number):void {
+  if (arg <= 0) {
+    throw new Error('One of side or radius <= 0');
+  }
 }
```

Comment:
Optional: you could rewrite `checkArg` function so that it will accept arguments (any length) and determine if some have `value <= 0`.
```
(...args) => args.some() ....
```